timelock-agent/src/main/java/com/palantir/timelock/paxos/TimeLockAgent.java

```diff
@@ -95,6 +101,12 @@
     private LeaderPingHealthCheck healthCheck;
     private TimelockNamespaces namespaces;
 
+    private final TimeLockClientFeedbackSink timeLockClientFeedbackSink = TimeLockClientFeedbackSink
```

Comment:
in terms of class design, I'd prefer to maintain a reference to the `FeedbackProvider` instead of the sink in this class (the idea being that users should talk to the provider, not retrieve the reports directly)